### PR TITLE
PS-2021 automate the build

### PR DESCRIPTION
**Pull request by pivnicek**
The pipeline is here https://dev.azure.com/keboola-dev/Sandboxes/_build?definitionId=74&_a=summary

I removed the travis from this branch so it seems it gets stuck on yellow check here...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,3 @@
-FROM quay.io/keboola/docker-custom-python:2.2.0
```
Comment (JakubMatejka) [inline Dockerfile:1]: Shouldn't we use the one in ACR?

Comment (pivnicek) [reply Dockerfile:1]: ok, changed it in the pipeline

Comment (pivnicek) [reply Dockerfile:1]: ah, sorry, needs an acr login first

Comment (pivnicek) [reply Dockerfile:1]: alright, that seems to work well 🍻

**Review by JakubMatejka — DISMISSED** [review]
Looks good. 👍 (or we see soon in the pipeline. 😀)

**Comment by pivnicek** [comment]
It should be good, I've had a non-semantic tag pass

**Comment by pivnicek** [comment]
sorry, was one last change needed

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**